docs/learn/smart-contract-developers/create-nft-collection.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: '🗃 Create an NFT Collection'
+sidebar_position: 4
+---
+
+# Create an NFT Collection Using LSP8
+
+This tutorial goes over creating a collection of collectibles, where each item is unique and has its own design.
+
+:::info
+⌨️ The full code of this example can be found in 👾 [here](https://github.com/CJ42/LUKSO-Hardhat-template).
+:::
+
+## Setup
+
+Make sure you have the following dependencies installed:
+
+- [`ethers.js`](https://github.com/ethers-io/ethers.js/) (alternatively you can use [`web3.js`](https://github.com/web3/web3.js))
+- [`@lukso/lsp-smart-contracts`](https://github.com/lukso-network/lsp-smart-contracts/)
+
+## Create the Smart Contracts
+
+When creating digital asset smart contracts on LUKSO, you will need to specify the token type and the following data keys in the ERC725Y storage. There are three types that you can define:
+
+- 0 = Token
+- 1 = NFT
+- 2 = Collection
+
+```solidity title="contracts/TokenTypes.sol"
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+bytes32 constant _LSP4_TOKEN_TYPE_DATA_KEY = 0xe0261fa95db2eb3b5439bd033cda66d56b96f92f243a8228fd87550ed7bdfdb3;
+
+enum TokenType {
+    TOKEN,
+    NFT,
+    COLLECTION
+}
+
+
+```
+
+You will also create a custom [LSP8 Identfiable Digital Asset Collection](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) that extends [LSP8Mintable](../../contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.md).
```

Comment:
```suggestion
After defining the type of the asset and its 🗂️ [ERC725 data key](../../standards/lsp-background/erc725.md#erc725y-generic-data-keyvalue-store) you can create a custom 🌄 [LSP8 Identfiable Digital Asset Collection](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) that extends [LSP8Mintable](../../contracts/contracts/LSP8IdentifiableDigitalAsset/presets/LSP8Mintable.md) so that new assets can be created within the smart contract.
```